src/control/lib/control/support.go

```diff
@@ -0,0 +1,69 @@
+//
+// (C) Copyright 2022-2023 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+
+package control
+
+import (
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"google.golang.org/protobuf/proto"
+
+	ctlpb "github.com/daos-stack/daos/src/control/common/proto/ctl"
+)
+
+type (
+	// CollectLogReq contains the parameters for a collect-log request.
+	CollectLogReq struct {
+		unaryRequest
+		TargetFolder string
+		TargetHost   string
+		ExtraLogsDir string
+		JsonOutput   bool
```

Comment:
Why does the server need to know about this client-side display option?

Reply:
We are passing the same option to dmg log collection. Admin has both the option to collect the logs with/without JSON. And support engineer also prefer the JSON in case we have a lot of data to analyze better.